### PR TITLE
Modern Checkmark and Info button & currency dropdown in the Loans dialog

### PR DESCRIPTION
**Pull request by jpdup**
No functional changes

### CODE REVIEW

File: Source/WPF/MyMoney/View Selectors/BalanceControl.xaml [hunk 1]
```diff
@@ -66,8 +66,8 @@ For example, you can use this to swap out a reconciled transaction for another o
                                     <ColumnDefinition Width="Auto"/>
                                     <ColumnDefinition Width="*"/>
                                 </Grid.ColumnDefinitions>
-                                <Button x:Name="PreviousBalanceHelp" Click="PreviousBalanceHelp_Click">
-                                    <Image Source="/MyMoney;component/Icons/Help.png" />
+                                <Button x:Name="PreviousBalanceHelp" Click="PreviousBalanceHelp_Click" Background="Transparent">
+                                    <Label>ℹ️</Label>
```
Comment (clovett) [inline Source/WPF/MyMoney/View Selectors/BalanceControl.xaml:70]: Doesn't scream "click here for more information" to me:
image

FontSize 16 is a bit better...
image

Comment (clovett) [reply Source/WPF/MyMoney/View Selectors/BalanceControl.xaml:70]: Is there a way to get a blue background like I see here?
https://emojipedia.org/information/

File: Source/WPF/MyMoney/Dialogs/LoanDialog.xaml.cs [hunk 2]
```diff
@@ -70,6 +71,10 @@ public LoanDialog(MyMoney money, Account a)
                     InsertAccount(oa);
                 }
             }
+
+            List<string> currencies = new List<string>(Enum.GetNames(typeof(RestfulWebServices.CurrencyCode)));
```
Comment (clovett) [inline Source/WPF/MyMoney/Dialogs/LoanDialog.xaml.cs:75]: Pity these don't have tooltips...

**Review by clovett — CHANGES_REQUESTED** [review]

**Comment by clovett** [comment]
Thanks for the PR.

File: Source/WPF/MyMoney/View Selectors/BalanceControl.xaml (hunk 1 above)
Comment (clovett) [reply Source/WPF/MyMoney/View Selectors/BalanceControl.xaml:70]: Waddya think about this version?
```
              <Button x:Name="CurrentDeltaHelp" Click="CurrentDeltaHelp_Click" Background="#3A86C2" Padding="0,0,0,1">
                <Label FontSize="18" Foreground="White" HorizontalAlignment="Center" VerticalAlignment="Center">ℹ️</Label>
              </Button>
```
image

Comment (clovett) [reply Source/WPF/MyMoney/View Selectors/BalanceControl.xaml:70]: image

Comment (clovett) [reply Source/WPF/MyMoney/View Selectors/BalanceControl.xaml:70]: I got the colors from Win+. emoji selector:

image

and we could templatize this button style on our themes so we can use it anywhere.

Comment (clovett) [reply Source/WPF/MyMoney/View Selectors/BalanceControl.xaml:70]: I'd prefer to get rid of the white border which seems to be coming from a unicode variation 0xFE0F, but I couldn't figure out how... Notice github shows the windows emoji like this: ℹ️

File: Source/WPF/MyMoney/Dialogs/LoanDialog.xaml.cs (hunk 2 above)
Comment (clovett) [reply Source/WPF/MyMoney/Dialogs/LoanDialog.xaml.cs:75]: I added the tooltips and templatized the style, and pushed this changes so I can merge the PR.

File: Source/WPF/MyMoney/View Selectors/BalanceControl.xaml (hunk 1 above)
Comment (jpdup) [reply Source/WPF/MyMoney/View Selectors/BalanceControl.xaml]: This all looks good the  way you are doing now. ship it

**Pull request merged**